requirements-tests.txt

```diff
@@ -7,3 +7,4 @@ pylint
 hypothesis
 hypothesis-networkx
 codecov
+vermouth
```

Comment:
Should not be needed, since it's required to install polyply itself. requirements-tests.txt should only list packages that are required additionally to run the tests.
Also, AFAIK your tests don't use hypothesis, so remove those from here.
And add pytest